client/src/App.js

```diff
@@ -1,21 +1,41 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { addPlayer, startGame } from './actions/hanabiActions'
+import Game from './components/Game'
 
 class App extends Component {
+  constructor(props){
+    super(props);
+  }
+  
   render() {
     return (
       <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+        <Game
+          startGame = {this.props.actions.startGame}
```

Comment:
this unneeded whitespace between `startGame` and `=`, in the JSX, can cause compiling errors in JavaScript.